doc/BUILD_ENVIRONMENT.md

```diff
@@ -2,7 +2,7 @@
 
 Building and testing CORTX can be done using a single system.  Obviously CORTX is designed to be a scale-out distributed object storage system but these instructions currently only cover setting up a single node environment.
 
-Currently CORTX requires the CentOS 7.7.1908 distribution running the 3.10.0-1062 kernel. Building it and testing it can be done on either a physical or virtual machine matching these requirements.  We have provided instructions to [install dependencies](InstallingDependencies.md).
+Currently CORTX requires the CentOS 7.8.2003 distribution running the 3.10.0-1062 kernel. Building it and testing it can be done on either a physical or virtual machine matching these requirements.  We have provided instructions to [install dependencies](InstallingDependencies.md).
 
```

Comment:
@Saumya-Sunder  Please update `3.10.0-1062` kernel to `3.10.0-1127` kernel.

Reply:
done